Assets/Scripts/Camera/AutoCameraFocus.cs

```diff
@@ -0,0 +1,19 @@
+using Cinemachine;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AutoCameraFocus : MonoBehaviour
+{
+    private CinemachineFreeLook _camera;
+
+    private void Start()
+    {
+        _camera = FindObjectOfType<CinemachineFreeLook>();
+
+        _camera.Follow = this.transform;
+        _camera.LookAt = this.transform;
+
+        Destroy(this);
```

Comment:
Is the camera target suppose to be destroyed immediately?

Reply:
Oh wait, I think I understand this now. This deletes the AutoCameraFocus component, not the actual game object.